src/ExNavigationBar.js

```diff
@@ -136,18 +136,20 @@ const buttonStyles = StyleSheet.create({
     }),
   },
   menuButton: {
-    height: 26,
-    width: 26,
+    resizeMode: 'contain',
     ...Platform.select({
       ios: {
-        margin: 10,
+        height: 26,
+        width: 26,
+        marginLeft: 8,
+        marginRight: 6,
       },
       android: {
-        marginLeft: 23,
-        marginTop: -1,
+        height: 24,
```

Comment:
on android menuButton and back button has same styles. so we are duplicating here. That's not true for ios tho.

I guess it's fine. gives us freedom to change it without breaking other styles.